barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.hpp

```diff
@@ -116,6 +116,7 @@ struct AcirFormat {
     // This could be a large vector so use slab allocator, we don't expect the blackbox implementations to be so large.
     bb::SlabVector<PolyTripleConstraint> poly_triple_constraints;
     bb::SlabVector<bb::mul_quad_<bb::curve::BN254::ScalarField>> quad_constraints;
```

Comment:
I think these need comments. Someone without context will spend at least 30 minutes trying to understand what's the point of these

Reply:
Done